wiki/Beatmap_Spotlights/zh.md

```diff
@@ -1,52 +1,51 @@
 ---
-no_native_review: true
-outdated: true
-outdated_since: 542060abc5b2a4178a1b8d4a897ccef0d7149c4d
 tags:
-  - Spotlights
-  - Charts
+  - Beatmap Spotlights
+  - charts
   - Ranking Charts
+  - Seasonal Spotlights
+  - Spotlights
   - 月赛
+  - 季赛
 ---
 
 # 月赛
 
-![Beatmap Spotlights Logo](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+![Beatmap Spotlights logo](/wiki/shared/news/banners/beatmap-spotlights.jpg)
 
-**月赛**（Beatmap Spotlights，也被称为 *Ranking Charts*）是一个定期举办的活动系统，用于推荐或突出玩家制作的拥有出色或独特玩法设计的谱面。它有一个赛季排行榜，并为所有参与的玩家提供奖励。
+**月赛**（Beatmap Spotlights，也被称为 *Ranking Charts*）是一个定期举办的活动系统，用于推荐或突出玩家制作的拥有出色或独特玩法设计的[谱面](/wiki/Beatmaps)。它有一个[赛季排行榜](#赛季)，并为所有参与的玩家提供奖励。
```

Comment:
"Beatmap Spotlights" should be italics, since you're pointing out a title for the first time